db/migrate/20170629203747_add_new_identifier_tables.rb

```diff
@@ -0,0 +1,20 @@
+class AddNewIdentifierTables < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rosters do |t|
+      t.string :identifier_name, null: false
+
+      t.timestamps null: false
+    end
+
+    create_table :roster_entries do |t|
+      t.string :identifier, null: false
+      t.references :roster, null: false
+      t.references :user
```

Comment:
Does a roster entry need to have a user? Since we don't specify I'm assuming not, but I just wanted to be sure.